charts/nginx-ingress/values.yaml

```diff
@@ -21,9 +21,65 @@ controller:
   appprotect:
     ## Enable the App Protect WAF module in the Ingress Controller.
     enable: false
+    ## Enables App Protect WAF v5.
+    v5: false
     ## Sets log level for App Protect WAF. Allowed values: fatal, error, warn, info, debug, trace
     # logLevel: fatal
 
+    # Volumes for App Protect WAF v5
+    # Required volumes are: app-protect-bd-config, app-protect-config, and app-protect-bundles
+    volumes:
+      - name: app-protect-bd-config
+        emptyDir: {}
+      - name: app-protect-config
+        emptyDir: {}
+      - name: app-protect-bundles
+        emptyDir: {}
+
+    ## Configuration for App Protect WAF v5 Enforcer
+    enforcer:
+      # Host that the App Protect WAF v5 Enforcer runs on.
+      # This will normally be "127.0.0.1" as the Enforcer container
+      # will run in the same pod as the Ingress Controller container.
+      host: "127.0.0.1"
+      # Port that the App Protect WAF v5 Enforcer runs on.
+      port: 50000
+      image:
+        ## The image repository of the App Protect WAF v5 Enforcer.
+        repository: private-registry.nginx.com/nap/waf-enforcer
+
+        ## The tag of the App Protect WAF v5 Enforcer image.
+        tag: "5.2.0"
+        ## The digest of the App Protect WAF v5 Enforcer image.
+        ## If digest is specified it has precedence over tag and will be used instead
+        # digest: "sha256:CHANGEME"
+
+        ## The pull policy for the App Protect WAF v5 Enforcer image.
+        pullPolicy: IfNotPresent
+      # securityContext: {}
+
+    ## Configuration for App Protect WAF v5 Configuration Manager
+    configManager:
+      image:
+        ## The image repository of the App Protect WAF v5 Configuration Manager.
+        repository: private-registry.nginx.com/nap/waf-config-mgr
+
+        ## The tag of the App Protect WAF v5 Configuration Manager image.
+        tag: "5.2.0"
+        ## The digest of the App Protect WAF v5 Configuration Manager image.
+        ## If digest is specified it has precedence over tag and will be used instead
+        # digest: "sha256:CHANGEME"
+
+        ## The pull policy for the App Protect WAF v5 Configuration Manager image.
+        pullPolicy: IfNotPresent
+      securityContext:
```

Comment:
@oseoin is this required or can be left empty:
```
  securityContext: {} # Remove curly brackets before adding values
    # allowPrivilegeEscalation: true
    # readOnlyRootFilesystem: true
    # runAsUser: 101 #nginx
    # runAsNonRoot: true
    # capabilities:
    #   drop:
    #   - ALL
    #   add:
    #   - NET_BIND_SERVICE
```

Reply:
I think these came from [here](https://docs.nginx.com/nginx-app-protect-waf/v5/admin-guide/deploy-on-kubernetes/) - @jjngx can you confirm if these came from testing?

Reply:
confirmed